src/Features/LanguageServer/Protocol/Features/CodeFixes/CodeFixService.cs

```diff
@@ -489,6 +489,7 @@ private bool TryGetWorkspaceFixersPriorityMap(TextDocument document, [NotNullWhe
                                 var fixerMetadata = TryGetMetadata(fixer);
 
                                 using (addOperationScope(fixerName))
+                                using (Logger.LogBlock(FunctionId.CodeFixes_GetCodeFixesAsync, KeyValueLogMessage.Create(LogType.UserAction, m => CreateLogProperties(m, fixer)), cancellationToken))
```

Comment:
consider moving this inside `TryGetFixesOrConfigurationsAsync`. Currently we are not awaiting the task so we imediately return.